File: Lehrverwaltung/server/src/main/java/tie/lehrlinge/lehrverwaltung/server/model/NotificationModel.java
package tie.lehrlinge.lehrverwaltung.server.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class NotificationModel {
    int status;

    String title;

    String description;
}

File: Lehrverwaltung/client/src/app/core/models/notification/NotificationModel.ts
import {STATUS} from "./STATUS";

export class NotificationModel {
  status: STATUS;
  title: string;

  description: string;

  constructor(status: STATUS, title: string, description: string) {
    this.status = status;
    this.title = title;
    this.description = description;
  }
}
